### PR TITLE
reversal algorithm

### PR DESCRIPTION
**Pull request by Sapna2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes

### Description

(Write your answer here.)
Reversal algorithm to rotate an array in java under the algorithms section.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#3640

### CODE REVIEW

**Comment by Sapna2001** [comment]
Replit link : https://replit.com/@Sapna2001/reversal

File: Java/Algorithms/reversalAlgorithm.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+public class ReversalAlgorithm {
```
Comment (HarshCasper) [inline Java/Algorithms/reversalAlgorithm.java:5]: Add a beginner friendly description

File: Java/Algorithms/reversalAlgorithm.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	}
+
+	private static void printArray(int[] array) {
+		for(int i : array) 
+			System.out.print(i+" ");
+		System.out.println();
+	}
+
+	public static void main(String[] args) {
+		int[] array = {1,2,3,4,5,6,7};
```
Comment (HarshCasper) [inline Java/Algorithms/reversalAlgorithm.java]: Use dynamic input

File: Java/Algorithms/reversalAlgorithm.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		// right reversal
+		rightReversal(array,k);
+		//print array
+		printArray(array);
+	}
+}
+
+// Output:
+// 3 4 5 6 7 1 2 -> after left rotation of original array
+// 1 2 3 4 5 6 7 -> after right rotation of left rotated array
```
Comment (HarshCasper) [inline Java/Algorithms/reversalAlgorithm.java]: Put in multi-line comments

Comment (HarshCasper) [inline Java/Algorithms/reversalAlgorithm.java]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Sapna2001** [comment]
@HarshCasper I have made the changes

**Review by ankitaggarwal23 — COMMENTED** [review]
Code plagiarized from [here](https://www.geeksforgeeks.org/program-for-array-rotation-continued-reversal-algorithm/).

**Pull request closed without merging**